queue/queue.py

```diff
@@ -1,5 +1,9 @@
+# import sys
+# sys.path.append('../linked_list')
+# from linked_list import LinkedList
 import sys
-sys.path.append('../linked_list')
+import os
+sys.path.append(f'{os.getcwd()}/linked_list')
```

Comment:
Good idea with using the os module's _getcwd_ function to run the files straight from the main directory.